README.md

```diff
@@ -1,7 +1,7 @@
 vcr
 ===
 
-[![Build](https://img.shields.io/travis/vcr/vcr/master.svg?style=flat-square)](https://travis-ci.org/vcr/vcr)
+[![CI status](https://github.com/vcr/vcr/actions/workflows/test.yml/badge.svg)](https://github.com/vcr/vcr/actions/workflows/test.yml)
```

Comment:
Not totally sure how this works. But this could potentially show any PRs CI result.. 

Been passing the branch query parameter to the badge icon: `https://github.com/vcr/vcr/actions/workflows/test.yml/badge.svg?branch=master`

Docs about this: https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter

Reply:
Thanks, good call!